### DISCUSSION TITLE
Sendgrid Integration

### DISCUSSION THREAD

**Discussion by coderreco** [swellstores/community#565] [Help] [answered]
I have enabled the Sendgrid integration with my API key, what's next? First time using Sendgrid but have built with Resend before. 

In regards to the Swell integration, now that I have added my API key and enabled, is there anything else I need to do to receive the notifications in Sendgrid? Will Swell now automatically post notifications to the Sendgrid endpoint and I just need to make my dynamic templates?

Thanks!

**Comment by logeshswell** [comment, marked as answer]
Yes, you are good to go.
Swell sends email notifications to your customers from your sendgrid account now.
For more details, please refer: https://www.swell.is/help/integrations/sendgrid

**Reply by coderreco** [reply]
Hi Logesh,

So, with the Sendgrid integration it just uses my Sendgrid account but uses the Swell templates? So in order to customize my emails I still use the notification templates in the swell dashboard?

[https://www.swell.is/help/integrations/sendgrid](https://www.swell.is/help/integrations/sendgrid) This link doesn't really give any detail on what it does, just how to add the API key.

Thanks for your help

**Reply by logeshswell** [reply]
> So, with the Sendgrid integration it just uses my Sendgrid account but uses the Swell templates? So in order to customize my emails I still use the notification templates in the swell dashboard?

Yes, you're right.
